File: app/src/main/java/com/pisces/android/wuha/mine/CallActivity.java
package com.pisces.android.wuha.mine;

import android.os.Bundle;
import android.support.annotation.Nullable;

import com.pisces.android.framworkerlibrary.core.LBaseActivity;
import com.pisces.android.wuha.R;

/**
 * Created by Chris Li on 2017/9/1.
 * 联系我们界面
 */

public class CallActivity extends LBaseActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_call);
        setToolbarTitle("联系我们");
    }
}

File: app/src/main/java/com/pisces/android/wuha/mine/CallActivity.kt
package com.pisces.android.wuha.mine

import android.os.Bundle

import com.pisces.android.wuha.R
import com.pisces.android.wuha.base.LBaseActivity

/**
 * Created by Chris Li on 2017/9/1.
 * 联系我们界面
 */

class CallActivity : LBaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_call)
        setToolbarTitle("联系我们")
    }
}
